CHANGELOG.rst

```diff
@@ -12,6 +12,8 @@ Improvements and new features:
 
 Documentation:
 
+ - Add error to STDOUT --fail-under-line or --fail-under-branch. (:issue:`502`)
```

Comment:
```suggestion
 - Add error to STDERR for --fail-under-line or --fail-under-branch. (:issue:`502`)
```

The error messages are actually written to STDERR, as it should be. If messages were written to STDOUT, this would also conflict with reports that are printed to STDOUT, e.g. `gcovr -x > cov.xml`.